app/views/admin/_import_inbox_step.erb

```diff
@@ -49,7 +49,7 @@
         All newcomers seem to have been assigned WCA IDs.
         Please <%= link_to "delete the inbox rows", competition_admin_delete_inbox_data_path(@competition, model: :inbox_person), class: 'inbox-trigger btn btn-primary', data: { jquery_method: :delete } %>.
       <% else %>
-        Please <%= link_to "assign WCA IDs to newcomers", admin_complete_persons_path(finish_persons_form: { competition_ids: @competition.id }), target: '_blank', class: 'btn btn-primary' %>
+        Please <%= link_to "assign WCA IDs to newcomers", admin_complete_persons_path(competition_ids: [@competition.id]), target: '_blank', class: 'btn btn-primary' %>
```

Comment:
Why do you need to wrap this in an array? Passing it as a string should be just fine

Reply:
Well, right here you're passing it to a `*_path` Rails helper, which encodes it into a URL anyways. So when reading it again in the controller, won't it end up as a string in each case?

Sorry my previous comment wasn't specific, but what I'm surprised about is the fact that you explicitly wrap it in an array _just before encoding it into a string URL_.

What is the difference in terms of the resulting URL anyways?